2020-02-Spring/3_trees/README.md

```diff
@@ -45,21 +45,41 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. Deepest Leaves Sum
 
-Source: TODO :bug:
+Source: https://leetcode.com/problems/deepest-leaves-sum/
```

Comment:
```suggestion
Source: [LeetCode](https://leetcode.com/problems/deepest-leaves-sum/)
```
Doing this would make this part look cleaner and more consistent.
To hyperlink something put [square brackets around phases needed to be linked](and the link goes here inside the parenthesis).

Reply:
Thanks! I have implemented this change. The new Readme is in the newest file committed